chart/Chart.yaml

```diff
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.7 
+version: 0.2.8
```

Comment:
If merged after #21 we need to bump the chart version to 0.2.10

Reply:
Resolved with https://github.com/viadee/docker-hub-rate-limit-exporter/pull/20/commits/3ced95af9e553c8e6dd1e52ebc7b6496d480be69